### DISCUSSION TITLE
OverwriteModelError: Cannot overwrite `File` model once compiled. While using webpack alias

### DISCUSSION THREAD

**Discussion by rubencosta13** [webpack/webpack#16560] [Q&A] [answered]
# Bug report

**What is the current behavior?**
```
/app/node_modules/mongoose/lib/index.js:553
       throw new _mongoose.Error.OverwriteModelError(name);
       ^
 
OverwriteModelError: Cannot overwrite `File` model once compiled.
     at Mongoose.model (/app/node_modules/mongoose/lib/index.js:553:13)
     at Object.535 (/app/main.js:1:1666)
     at r (/app/main.js:1:7956)
     at Object.484 (/app/main.js:1:2285)
     at r (/app/main.js:1:7956)
     at Object.945 (/app/main.js:1:198)
     at r (/app/main.js:1:7956)
     at /app/main.js:1:8005
     at Object.<anonymous> (/app/main.js:1:8943)
```

**If the current behavior is a bug, please provide the steps to reproduce.**

When using alias on webpack:
```
alias: {
      '@/': path.resolve(__dirname, 'src/'),
      'routes': path.resolve(__dirname, 'src/routes/'),
      'models': path.resolve(__dirname, 'src/models/'),
    }
```
And changing a line on my node.js project from 
``import { fileAdd } from './routes/add';``
to 
``import { fileAdd } from 'routes/add';``
I get that output (error) pasted in the current behavior

**What is the expected behavior?**
The expected behavior would be the app to instead of displaying that error would display "App started". Starting my expressjs application

**Other relevant information:**
webpack version:  "^5.75.0",
Node.js version: "v16.16.0"
Operating System: "MAC OSX"
Additional tools: "Docker & Kubernetes" 

**P.S** Let me know if there's anything missing or you need anything else

**Comment by alexander-akait** [comment, marked as answer]
Because in `'routes': path.resolve(__dirname, 'src/routes/')` `routes` is RegExp, so you mostly override internal `import`/`require`, I don't recommend to use `routes` or `models` without special characters like `@`, because you can `alias` unexpected things

**Reply by rubencosta13** [reply]
Just worked! Thank you very much!
